include/range/v3/action/action.hpp

```diff
@@ -132,7 +132,7 @@ namespace ranges
                 CONCEPT_REQUIRES_(is_pipeable<Action>() && Range<Rng &>() &&
                     Invocable<bitwise_or, ref_t<Rng &>, Action &>() &&
                     Same<ref_t<Rng &>,
-                        result_of_t<bitwise_or(ref_t<Rng &> &&, Action &)>>())>
+                        invoke_result_t<bitwise_or, ref_t<Rng &> &&, Action &>>())>
```

Comment:
`ref_t<Rng &> &&` could be simplified to `ref_t<Rng &>` here (because `declval`).